debian/patches/specify-max-resolution.patch

```diff
@@ -5,44 +5,109 @@ In the change introduced by https://codereview.chromium.org/872623002, Chromium
++
++void VASetFallbackResolution(gfx::Size* resolution)
++{
++  resolution->SetSize(va_fb_res_width, va_fb_res_height);
++}
++
+ bool VaapiWrapper::GetMaxResolution_Locked(
+     VAProfile va_profile,
+     VAEntrypoint entrypoint,
+@@ -460,7 +474,11 @@ bool VaapiWrapper::GetMaxResolution_Lock
    unsigned int num_attribs;
    va_res = vaQuerySurfaceAttributes(va_display_, va_config_id, nullptr,
                                      &num_attribs);
 -  VA_SUCCESS_OR_RETURN(va_res, "vaQuerySurfaceAttributes failed", false);
 +  if (va_res != VA_STATUS_SUCCESS) {
-+    LOG_VA_ERROR_AND_REPORT(va_res, "vaQuerySurfaceAttributes failed, returning hardcoded resolution");
-+    resolution->SetSize(1920, 1088); // Yes, this should be 1088.
++    LOG_VA_ERROR_AND_REPORT(va_res, "vaQuerySurfaceAttributes failed, returning fallback resolution");
++    VASetFallbackResolution(resolution);
 +    return true;
 +  }
    if (!num_attribs)
      return false;
  
-@@ -469,7 +473,11 @@ bool VaapiWrapper::GetMaxResolution_Lock
+@@ -469,7 +487,11 @@ bool VaapiWrapper::GetMaxResolution_Lock
  
    va_res = vaQuerySurfaceAttributes(va_display_, va_config_id, &attrib_list[0],
                                      &num_attribs);
 -  VA_SUCCESS_OR_RETURN(va_res, "vaQuerySurfaceAttributes failed", false);
 +  if (va_res != VA_STATUS_SUCCESS) {
-+    LOG_VA_ERROR_AND_REPORT(va_res, "vaQuerySurfaceAttributes failed, returning hardcoded resolution");
-+    resolution->SetSize(1920, 1088); // Yes, this should be 1088.
++    LOG_VA_ERROR_AND_REPORT(va_res, "vaQuerySurfaceAttributes failed, returning fallback resolution");
++    VASetFallbackResolution(resolution);
 +    return true;
 +  }
  
    resolution->SetSize(0, 0);
    for (const auto& attrib : attrib_list) {
-@@ -479,9 +487,7 @@ bool VaapiWrapper::GetMaxResolution_Lock
+@@ -479,9 +501,7 @@ bool VaapiWrapper::GetMaxResolution_Lock
        resolution->set_height(attrib.value.value.i);
    }
    if (resolution->IsEmpty()) {
 -    LOG(ERROR) << "Codec resolution " << resolution->ToString()
 -               << " cannot be zero.";
 -    return false;
-+    resolution->SetSize(1920, 1088); // Yes, this should be 1088.
++    VASetFallbackResolution(resolution);
    }
    return true;
  }
+@@ -1126,6 +1146,36 @@ void VaapiWrapper::DeinitializeVpp() {
+ 
+ // static
+ void VaapiWrapper::PreSandboxInitialization() {
++  int w=VA_FB_RES_DEFAULT_WIDTH, h=VA_FB_RES_DEFAULT_HEIGHT;
++  std::string VA_FALLBACK_RESOLUTION_FILENAME("/etc/chromium-browser/va_fallback_resolution");
++  std::string line;
++  // Open fallback resolution file before sandbox initialization
++  std::ifstream va_fb_res_file(VA_FALLBACK_RESOLUTION_FILENAME);
++  // If the file exists, open and read the resolution
++  // If not successful, use default values
++  if(va_fb_res_file.is_open()){
++    std::cout << "Opened fallback resolution file: " << VA_FALLBACK_RESOLUTION_FILENAME << "\n";
```

Comment:
Can you replace `std::cout` with either `LOG(ERROR)`, `LOG(WARNING)`, or `LOG(INFO)`? (Whichever one of those is appropriate) See surrounding methods for examples on usage.